subsys/bluetooth/host/smp.c

```diff
@@ -3930,6 +3951,48 @@ static uint8_t smp_ident_info(struct bt_smp *smp, struct net_buf *buf)
 	return 0;
 }
 
+static uint8_t smp_id_add_replace(struct bt_smp *smp, struct bt_keys *new_bond)
+{
+	struct bt_keys * conflict;
+
+	/* Sanity check: It does not make sense to finalize a bond before we
+	 * have the remote identity.
+	 */
+	__ASSERT_NO_MSG(!(smp->remote_dist & BT_SMP_DIST_ID_KEY));
+
+	conflict = bt_id_find_conflict(new_bond);
+
```

Comment:
You can remove this line (generally: no empty line between assignment to a variable and testing for its value)